src/main/java/com/takipi/oss/benchmarks/jmh/loops/ByteArrayLoopBenchmark.java

```diff
@@ -0,0 +1,107 @@
+package com.takipi.oss.benchmarks.jmh.loops;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Random;
+import java.util.concurrent.TimeUnit;
+import org.openjdk.jmh.annotations.Benchmark;
+import org.openjdk.jmh.annotations.BenchmarkMode;
+import org.openjdk.jmh.annotations.Fork;
+import org.openjdk.jmh.annotations.Measurement;
+import org.openjdk.jmh.annotations.Mode;
+import org.openjdk.jmh.annotations.OutputTimeUnit;
+import org.openjdk.jmh.annotations.Param;
+import org.openjdk.jmh.annotations.Scope;
+import org.openjdk.jmh.annotations.Setup;
+import org.openjdk.jmh.annotations.State;
+import org.openjdk.jmh.annotations.Warmup;
+
+@State(Scope.Benchmark)
+@BenchmarkMode(Mode.AverageTime)
+@OutputTimeUnit(TimeUnit.MILLISECONDS)
+@Fork(2)
+@Measurement(iterations = 5)
+@Warmup(iterations = 5)
+public class ByteArrayLoopBenchmark {
+    int size = 100_000;
+
+    int byteArrLength = 100;
+
+    List<byte[]> arrays;
+
+    @Setup
+    public void setup() {
+        arrays = new ArrayList<>(size);
+
+        Random random = new Random();
```

Comment:
The original test didn't provide a seed, so I didn't care to change that without a good reason. That said, if you can provide a good motivation, I'm game... What effect do you see providing a seed having? Either way, Arrays.hashCode() will have to traverse the full contents of every byte[], and the math in Math.max() will be unpredictable no matter what. If your point is simply "it could be predictable, but it's not right now", I agree with that -- I just don't see it actually having much of an effect, especially given that this setup runs once per iteration, not once per trial, so from the CPU's perspective it's going to run for a few billion cycles on the exact same set of data each time.